server/pulp/plugins/util/saxwriter.py

```diff
@@ -0,0 +1,94 @@
+from xml.sax.handler import ContentHandler
+from xml.sax.saxutils import escape, quoteattr
+
+
+class XMLWriter(ContentHandler):
+    """
+    XML writer similar to xml.sax.saxutils.XMLGenerator.
+    The result is indented XML, which is written sequentially to the stream.
```

Comment:
Since you also added the ``short_empty_elements`` feature, it's definitely worth mentioning here.